frontend/src/components/Login.js

```diff
@@ -56,7 +56,7 @@ class Login extends React.Component {
                                 name='username'
                                 placeholder='Username or e-mail'
                                 type='text'
-                                autoFocus=''
+                                autofocus="true"
```

Comment:
Prefer `autofocus={true}`.  Also, shouldn't the react property be `autoFocus` (the HTML property is `autofocus` I think)...

Reply:
I think that's right. I have updated `autofocus` to be `autoFocus`